File: src/pl/funnyskaddon/core/fix/GuildCreateListener.java
package pl.funnyskaddon.core.fix;

import org.bukkit.Bukkit;
import org.bukkit.event.EventHandler;
import org.bukkit.event.Listener;

import net.dzikoysk.funnyguilds.event.guild.GuildCreateEvent;
import pl.funnyskaddon.core.FunnySkAddon;

public class GuildCreateListener implements Listener{
	
	@EventHandler
	public void onGuildCreate(final GuildCreateEvent e) {
		Bukkit.getServer().getScheduler().runTaskLater(FunnySkAddon.getInst(), new Runnable() {
			@Override
			public void run() {
				final GuildCreateEventFix event = new GuildCreateEventFix(e.getDoer().getPlayer(), e.getTag());
				Bukkit.getServer().getPluginManager().callEvent(event);
			}
		}, 2);
	}
	
}

File: src/main/kotlin/pl/funnyskaddon/events/guilds/GuildCreateListener.kt
package pl.funnyskaddon.events.guilds

import net.dzikoysk.funnyguilds.event.guild.GuildCreateEvent
import org.bukkit.event.EventHandler
import org.bukkit.event.Listener
import pl.funnyskaddon.FunnySkAddon

class GuildCreateListener(val plugin: FunnySkAddon) : Listener {

    @EventHandler
    fun onCreate(event: GuildCreateEvent) {
        plugin.server.scheduler.runTaskLater(plugin, {
            plugin.server.pluginManager.callEvent(CustomGuildCreateEvent(event.doer.player, event.guild))
        }, 2)
    }

}